index.js

```diff
@@ -41,26 +40,26 @@ program
 console.log('Open Translators to Things CLI:');
 console.log('');
 
-if(program.menu) {
+if (program.menu) {
     prompt({ currentController: new MainController(), controllerStack: [] });
 }
 
 else if (program.onboarding) {
-    if (program.name === undefined) {
-        console.log("Need to provide name, -n <name>");
+    if (program.hub === undefined) {
+        console.log("Need to provide hub name, -h <hub name>");
         return;
     }
 
     console.log("------ Doing onboarding for %j".header, program.onboarding);
```

Comment:
Yup i agree --Could you file a tracking issue? thanks!

---
In reply to: [103779014](https://github.com/openT2T/opent2t-cli/pull/24#discussion_r103779014) [](ancestors = 103779014,103610447)